randr/rrcrtc.c

```diff
@@ -1217,9 +1217,6 @@ ProcRRGetCrtcInfo(ClientPtr client)
         }
     }
 
-    if (rpcbuf.error)
-        return BadAlloc;
-
```

Comment:
Maybe keep it to avoid calling `rrCrtcGet` below when there’s an error? (Not sure if that’s important though)